.github/workflows/4_bump_LSP_in_extension.yml

```diff
@@ -0,0 +1,40 @@
+name: Bump LSP in VSCode extension
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
+      branch:
+        description: 'Branch to run run this workflow on'
+        required: true
+
+jobs:
+  bump:
+    name: Bump LSP in VSCode extension
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      - name: Bump LSP version in extension
+        run: |
+            jq ".dependencies[\"@prisma/language-server\"] = \"${{ github.event.inputs.extension_version }}" \
+            ./packages/vscode/package.json >./packages/vscode/package.json.bk
+            mv ./packages/vscode/package.json.bk ./packages/vscode/package.json
```

Comment:
No in place editing possible?
(At least that is how I understand what this code is doing?)

Reply:
No, since only a part of it is modified, but I changed this to a JS file, which is a lot more readable.

Reply:
Still seeing this so probably not pushed yet.

Reply:
Ah I did create one for the other package json modifications, added one for the LSP now as well.